### DISCUSSION TITLE
Możliwość sprawdzenia metod nadania paczki dla metody dostawy

### DISCUSSION THREAD

**Discussion by czlowiekpolar** [allegro/allegro-api#8734] [Suggestions - resolved]
Proszę o udostępnienie przez API możliwości sprawdzenia dla jakiej metody dostawy udostępniana jest zmiana sposobu nadania paczki z "Zamówię podjazd kuriera" na "Nadam paczkę w Paczkomacie" lub "Nadam paczkę w PaczkoPuncie" oraz jakie sposoby nadania są dostępne dla danej metody dostawy. W chwili obecnej nie ma takiej możliwość i trzeba bazować na konkretnych identyfikatorach metod dostawy.

**Comment by MartaNowaczyk** [comment]
Wdrożone https://developer.allegro.pl/news/wysylam-z-allegro-udostepnilismy-mozliwosc-zamawiania-przesylek-inpost-z-uzyciem-metod-nadania-2YDanEB0rtD
